graphql_compiler/schema/schema_info.py

```diff
@@ -118,6 +123,9 @@ def make_sqlalchemy_schema_info(schema, type_equivalence_hints, dialect, tables,
         dialect: sqlalchemy.engine.interfaces.Dialect
         tables: dict mapping every graphql object type or interface type name in the schema to
                 a sqlalchemy table
+        primary_keys: dict mapping every graphql object type or interface type name in the schema
```

Comment:
Does the SQLAlchemy table object not have a notion of what its primary key is? I find that a bit surprising.

If that is indeed the case, and you decide to go with this approach, then let's validate that the columns specified here actually exist in the tables given in `tables`. If you also choose to allow composite primary keys, then the list of columns in such a composite key must be non-empty.

Reply:
It does. I went for an explicit external definition because we already ignore the `Table` object's information on foreign keys. If you think there's no reason to have implicit primary keys without formally declaring them as primary keys in your database, I'll switch to using the `Table`-defined primary keys.

Reply:
Reverted to using the primary key info from the sqlalchemy `Table`s, and documented that we expect tables to have primary keys.